### ISSUE TITLE
Unloading gallon jugs of milk produces "gallon jugs" instead of "unsealed gallon jugs"

### ISSUE THREAD

**Issue by Green-Greens** [CleverRaven/Cataclysm-DDA#44620]
# Describe the bug

Unloading gallon jugs of milk produces "gallon jugs" instead of "unsealed gallon jugs" which are needed in crafting recipes such as "makeshift funnel" 

# Steps To Reproduce

1. Find a gallon jug of milk (sealed), they commonly spawn in refrigerators in houses.
2. Unload the gallon jug of milk (sealed) onto the floor.
3. Look in your inventory and find a "gallon jug" rather than an "unsealed gallon jug"

# Expected behavior

With the current container system, unloading sealed containers should produce empty unsealed containers.

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1909
- Game Version: 0.E-6496-gef5a06d [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Blaze Industries [blazeindustries],
    Zetsukaze's Hair - 0.E EDIT [zets_hair_extensions_undead],
    Blaze Industries - UndeadPeople Tileset [BlazeIndustriesUDP],
    Whaleys Hair Mod (UDP Tweaks) [WHALEYHAIR_UDP]
]

**Comment by wapcaplet** [comment]
Confirmed in 0.E-6518-g8e6077ef45

**Comment by Qrox** [comment]
See #44359. "Unsealed gallon jug" is in fact the original "gallon jug" which cannot be sealed. "gallon jug" is now another kind of gallon jug that has been factory-sealed. So this is more like confusing naming rather than a bug. There was a discussion of better names in the original PR, but we didn't come up anything not too wordy and yet informative.

**Comment by Green-Greens** [comment]
Oh, that makes sense. Thank you for clarifying.

However, this means that there is at least one recipe that should include "gallon jug" as a substitutable component. The reason I noticed the original issue was because I wanted to make a makeshift funnel which requires "1x unsealed gallon jug OR 3x unsealed plastic bottle OR 6x small plastic bottle OR 6x aluminum foil". Should this list not include 1x gallon jug?

**Comment by Qrox** [comment]
I'm now thinking we should really merge the sealed and unsealed variants. The small difference of whether the bottle once had a factory seal doesn't really merit a different item type. I'm working on a fix but that would require some json adjustion.
